### DISCUSSION TITLE
any plans to support interactive theorem provers like coq and agda?

### DISCUSSION THREAD

**Discussion by SomeGuyNamedMay** [helix-editor/helix#6499] [General]
just wondering

**Comment by Rudxain** [comment]
[Coq even has an official LSP!](https://github.com/coq/coq/wiki/Configuration-of-Coq-LSP). But it's not configured as default in this repo's `languages.toml`

**Comment by HiiGHoVuTi** [comment]
this would really help me in my day to day life

**Comment by HiiGHoVuTi** [comment]
I did just create a [PR for coq](https://github.com/helix-editor/helix/pull/12202) using the LSP @Rudxain suggested.
